app/feedperson/tests.py

```diff
@@ -14,16 +14,29 @@ def test_str_value(self):
     feed_person = FeedPerson.objects.get(id=1)
     self.assertEqual(feed_person.name, str(feed_person))
 
-  def test_load_feed_people_removes_original_data(self):
-    feed_person = FeedPerson.objects.get(id=1)
+class LoadFeedTest(TestCase):
+  
+  @classmethod
+  def setUpTestData(cls):
     with disable_auto_indexing():
       load_feed_people()
-    self.assertEqual(FeedPerson.objects.filter(eppn=feed_person.eppn).count(), 0)
 
   def test_load_feed_people_imports_western_ave_people(self):
-    with disable_auto_indexing():
-      load_feed_people()
     western_ave_people = filter(lambda person: "Western Ave" in person.location, FeedPerson.objects.all())
     non_western_ave_people = filter(lambda person: "Western Ave" not in person.location, FeedPerson.objects.all())
     self.assertTrue(len(list(western_ave_people)) > 0)
     self.assertEqual(len(list(non_western_ave_people)), 0)
+
+  def test_load_feed_people_regex_removes_all_SciEnG(self):
+    scieng_people = filter(lambda person: "Sci&Eng" in person.location, FeedPerson.objects.all())
```

Comment:
In addition to testing that "Sci&Eng" is not present, would it make sense to test that SEC **does** appear?

Reply:
I thought about this, but SEC would be present even if the regex didn't work, as some of the entries already have that (e.g. Marie Dooley). I guess if it isn't there then something has definitely gone wrong, so maybe worth it anyway.

Reply:
Added test in bb38473